src/components/AssetDownload.tsx

```diff
@@ -1,13 +1,17 @@
 import React from "react"
 import styled from "@emotion/styled"
 import { GatsbyImage } from "gatsby-plugin-image"
-
+import { useEffect, useState } from "react"
+import * as ReactDOMServer from "react-dom/server"
+import Button from "./Button"
 import Translation from "../components/Translation"
 import ButtonLink from "./ButtonLink"
 import Emoji from "./OldEmoji"
 import Link from "./Link"
 
 import { getImage, getSrc, ImageDataLike } from "../utils/image"
+import { Box, Flex, Wrap, WrapItem } from "@chakra-ui/react"
+import theme from "../@chakra-ui/gatsby-plugin/theme"
```

Comment:
Some of these imports aren't used. Lets remove them.

And external lib imports should be imported at the top of the file (along with the react, react-dom import).